### PR TITLE
Feat added authentication pages

### PR DESCRIPTION
**Pull request by AmishMishra11**
- Added Login Page
- Added SignUp Page
- Added Wishlist Page
- Added Cart Page
- Added Routing
- Some minor styling changes

https://user-images.githubusercontent.com/76205249/159141512-187ee214-20c1-49c7-9813-01e2477632fd.mp4

### CODE REVIEW

File: src/Components/AuthCart/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        {/* <!-- items --> */}
+
+        <div className="flex-c">
+          {arr.map((item) => (
+            <CartCard key={item._id} item={item} />
+          ))}
+        </div>
+
+        {/* <!-- order --> */}
+        <div class="order border-radius-S flex-c">
```
Comment (ArjunGTX) [inline src/Components/AuthCart/index.js]: `class` can be replaced with `className`.

File: src/Components/AuthLogin/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import "./styles.css";
+import { Link } from "react-router-dom";
+
+function AuthLogin() {
+  return (
+    <div class="container-login flex-c">
+      <h1>Login</h1>
+
+      <div class="email">
+        <label for="email-id"> Email address</label>
```
Comment (ArjunGTX) [inline src/Components/AuthLogin/index.js]: Use `htmlFor` instead of `for`

**Review by ArjunGTX — COMMENTED** [review]
VsCode tip: use `ctrl` + `f2` to change all occurrences at once

File: src/Components/Category/index.jsx [hunk 3]
```diff
@@ -18,43 +18,41 @@ function Category() {
...
       <header className="category">
         <Link className="content flex-c" to="/products">
-          <div
-            className="color-box"
-            onClick={() => dispatchFilter({ type: "MEN", payload: !men })}
-          >
-            <img alt="Men" src={Men} />
+          <div onClick={() => dispatchFilter({ type: "MEN", payload: !men })}>
+            <div className="color-box">
+              <img alt="Men" src={Men} />
```
Comment (Mohitkumar33) [inline src/Components/Category/index.jsx:23]: in the alt you can directly call the gender from the database.

**Review by Mohitkumar33 — APPROVED** [review]

File: src/Components/AuthCart/index.js (hunk 1 above)
Comment (AmishMishra11) [reply src/Components/AuthCart/index.js]: Thank you for pointing it out i will fix that

File: src/Components/AuthLogin/index.js (hunk 2 above)
Comment (AmishMishra11) [reply src/Components/AuthLogin/index.js]: Thank you for pointing it out i will fix that

File: src/Components/Category/index.jsx (hunk 3 above)
Comment (AmishMishra11) [reply src/Components/Category/index.jsx:23]: Solved via voice call

**Review by ArjunGTX — APPROVED** [review]

**Pull request merged**